docs/extensibility/50-list-and-details-widgets.md

```diff
@@ -578,6 +581,64 @@ See the following example:
 
 <img src="./assets/display-widgets/ResourceRefs.png" alt="Example of a ResourceRefs widget" style="border: 1px solid #D2D5D9">
 
+### StatisticalCard
+
+StatisticalCard widgets render a card component with a several numerical information elements.
+This widget is primarily designed to be used in Monitoring and Health section **data.details.health** or via [injections](#widget-injections-overview) (**destination: ClusterOverview, slot: health**), allowing the card to be rendered within the dense grid layout of the ClusterOverview's Monitoring and Health section.
```

Comment:
```suggestion
To display the widget in the **Monitoring and Health** section of a details page, configure it in **data.details.health**.  
To render the card within the dense grid layout in the **Monitoring and Health** section of **Cluster Details**, use [injections](#widget-injections-overview) (**destination: ClusterOverview, slot: health**).
```

Reply:
![Screenshot 2024-06-21 at 11 56 59](https://github.com/kyma-project/busola/assets/23300533/08ea4b83-5124-4b76-aea0-a3bd750ae2c5)
Is it Cluster Overview or Cluster Details? PTAL at the screenshot.

Reply:
In navigation it's Cluster Details